_studio/mfx_lib/encode_hw/hevc/agnostic/g9/hevcehw_g9_roi.cpp

```diff
@@ -107,6 +107,10 @@ mfxStatus ROI::CheckAndFixROI(
 
     auto IsInvalidRect = [](const ROI::RectData& rect)
     {
+        if (rect.Top == 0 && rect.Left == 0 && rect.Right == 0 && rect.Bottom == 0)
+        {
+            return false;
+        }
         return ((rect.Left >= rect.Right) || (rect.Top >= rect.Bottom));
```

Comment:
Why are we using ">=" instead of ">" here?
Left==Right and Top==Buttom is a valid case, isn't it?